deployments/typesMillau.json

```diff
@@ -0,0 +1,52 @@
+{
+	"AuthoritySet": {
+		"authorities": "AuthorityList",
+		"set_id": "SetId"
+	},
+	"BridgedBlockHash": "H256",
+	"BridgedBlockHasher": "BlakeTwoAndKeccak256",
+	"BridgedBlockNumber": "u64",
+	"BridgedDigest": {
+		"logs": "Vec<BridgedDigestItem>"
+	},
+	"BridgedDigestItem": {
+		"_enum": {
+			"AuthoritiesChange": "Vec<AuthorityId>",
+			"ChangesTrieRoot": "BridgedBlockHash",
+			"Consensus": "Consensus",
+			"Other": "Vec<u8>",
+			"PreRuntime": "PreRuntime",
+			"Seal": "Seal",
+			"SealV0": "SealV0"
+		}
+	},
+	"BridgedHeader": {
+		"digest": "BridgedDigest",
+		"extrinsics_root": "BridgedBlockHash",
+		"number": "Compact<BridgedBlockNumber>",
```

Comment:
Why's this need to be `Compact`?

Reply:
@tomusdrw , same

Reply:
Because that's part of encode/decode implementation: https://github.com/paritytech/substrate/blob/master/primitives/runtime/src/generic/header.rs#L96

I basically took the built-in `Header` type from [here](https://github.com/polkadot-js/api/blob/d2ff350db963600fb5256152ef6946856e1bbb96/packages/types/src/interfaces/runtime/definitions.ts#L72)
and redefined the parts that depend on the hash size.